docs/storage/delta_format.md

```diff
@@ -0,0 +1,55 @@
+# Delta Format
+
+[Delta format][delta-io] is an open-source storage layer that brings ACID
+transaction properties to Apache Spark and other blob storage systems.
+
+Using the cloud-storage-etl-udfs, it is now possible to import data from the
+Delta format.
+
+## Import Delta formatted data
+
+Like other cloud storage systems, you can run the Exasol IMPORT SQL statement to
+import the data from the Delta format.
+
+Here is an example to import delta formatted data from Amazon S3:
+
+```sql
+CREATE OR REPLACE CONNECTION S3_CONNECTION
+TO ''
+USER ''
+IDENTIFIED BY 'S3_ACCESS_KEY=<AWS_ACCESS_KEY>;S3_SECRET_KEY=<AWS_SECRET_KEY>';
+
+IMPORT INTO RETAIL.SALES_POSITIONS
+FROM SCRIPT ETL.IMPORT_PATH WITH
+  BUCKET_PATH     = 's3a://<BUCKET>/import/delta/sales_positions/*'
+  DATA_FORMAT     = 'DELTA'
+  S3_ENDPOINT     = 's3.<REGION>.amazonaws.com'
+  CONNECTION_NAME = 'S3_CONNECTION'
+  PARALLELISM     = 'nproc()*2';
```

Comment:
Maybe we need a short comment about  PARALLELISM here? Can people change this value?

Reply:
It is described [here](https://github.com/exasol/cloud-storage-etl-udfs/blob/e5e11b661118bc9b60b571b1d3084282efe2ed7d/docs/user_guide.md#optional-properties). It is also described in the coming delta related blog.
Yes, it can be changed and also removed then the default `nproc()` is used.